storage/elasticsearch/elasticsearch.go

```diff
@@ -101,8 +99,7 @@ type ElasticSearchStorage struct {
 }
 
 func (c *ElasticSearchStorage) StoreFlows(flows []*flow.Flow) error {
-	c.indexer.Start()
-	defer c.indexer.Stop()
+	defer c.indexer.Flush()
```

Comment:
mmm, Flush is called when Stop, so no need to call it. I hesitated to remove the Flush but I thought you wanted to have something synchronous as you used Stop! finally bulk doesn't mean async :). I will remove it anyway